common/views/components/Pagination/Pagination.tsx

```diff
@@ -4,96 +4,117 @@ import Control from '../Buttons/Control/Control';
 import Space from '../styled/Space';
 import Rotator from '../styled/Rotator';
 import { arrow } from '@weco/common/icons';
+import styled from 'styled-components';
 
 export type Props = {
-  total: number;
-  prevPage?: number;
+  totalResults: number;
   currentPage: number;
-  pageCount: number;
+  totalPages: number;
+  prevPage?: number;
   nextPage?: number;
-  nextQueryString?: string;
   prevQueryString?: string;
+  nextQueryString?: string;
   range?: {
     beginning: number;
     end: number;
   };
 };
 
+const PaginatorContainer = styled(Space).attrs({
+  className: font('intr', 5),
+  v: {
+    size: 'm',
+    properties: ['padding-top', 'padding-bottom'],
+    overrides: { small: 5, medium: 5, large: 1 },
+  },
+})`
+  float: right;
```

Comment:
If it were `display: flex` rather than `inline-flex`, does that get rid of the need for the `float: right`?

Reply:
great shout, the `float` isn't even needed! `inline-flex` and `flex` did the same job here as the parent also has `display:flex`, so I changed it to simplify.